src/subcommands/server.go

```diff
@@ -21,36 +21,59 @@ func SemantiflyNewServer(serverIndexPath string) *Server {
 }
 
 func (s *Server) Add(ctx context.Context, req *pb.AddRequest) (*pb.AddResponse, error) {
+	db_ctx, conn, err := setupDBConn()
```

Comment:
This should be done when starting the server IIUC, not for each request

Reply:
Oh right. I will move it out